NETOBSERV-1610: document cardinality warning per field (#623)
netobserv/network-observability-operator: docs/flows-format.adoc

The "Loki label" column is useful when querying Loki directly: label fields need to be selected using link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].

> The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.

Author: cc @skrthomas for reviewing this part

> The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.

Reviewer: ```suggestion
The "Cardinality" column gives information about the implied metric cardinality if this field is used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.
```
